data/strategies/250/BeethovenJointProvider.json

```diff
@@ -1,13 +1,13 @@
 {
   "$schema": "strategy",
-  "name": "Beethoven-x Joint Provider",
-  "description": "Supplies {{token}} as half (or a third) of a joint liquidity provider for a liquidity pool on [Beethoven-x](https://beets.fi). Rewards are harvested, divided between the two (or three) pairs, sold for more {{token}}, and deposited back into the strategy.",
+  "name": "Beethoven-x Liquidity Boot Strapping Pool Joint Provider",
```

Comment:
Might be too long for UI – will break the lay out.

Reply:
Ok will fix yah this is a deprecated strategy and probably won't be used again, wasn't fully sure if I should add it but decided it might be best to do so

Reply:
Fixed